### ISSUE TITLE
replace files during protoc install under travis CI fails

### ISSUE THREAD

**Issue by koshatul** [make-files/adr#39]
```
Archive:  protoc-3.11.4-linux-x86_64.zip
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [{ENTER}]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [# Finally]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [, invoke ]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [the build]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [-resource]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [ command ]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [to produc]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [e the act]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [ual resou]
replace include/google/protobuf/type.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: new name:   inflating: # was requested.        
replace include/google/protobuf/duration.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [.makefile]
replace include/google/protobuf/duration.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [s/lib/cor]
replace include/google/protobuf/duration.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [e/bin/bui]
replace include/google/protobuf/duration.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [ld-resour]
replace include/google/protobuf/duration.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename: error:  invalid response [ce "$1"]
replace include/google/protobuf/duration.proto? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)
```

**Comment by jmalloc** [comment]
This should probably just test if protoc is already installed, and not install it at all.

**Comment by jmalloc** [comment]
I've merged the related PR, but created #40 to consider how this should actually behave.
